.github/workflows/release-jetbrains.yml

```diff
@@ -0,0 +1,80 @@
+name: release-jetbrains
+on:
+  push:
+    tags:
+      - "jetbrains/v*"
```

Comment:
I'd like to kick the pipeline when a release with jetbrains/vxxx tag is created.